frontend/src/lib/components/instances/ListView.svelte

```diff
@@ -97,7 +97,10 @@
 	}
 </script>
 
-<div class="overflow-y-auto flex flex-wrap content-start w-full h-full">
+<div
+	class="overflow-y-auto w-full h-full grid"
+	style="grid-template-columns: repeat(auto-fill, minmax(400px, 1fr))"
+>
```

Comment:
Looks awesome on my big screen, but on my integrated one somewhat off:
<img width="1622" alt="image" src="https://github.com/zeno-ml/zeno-hub/assets/5690524/46c54503-c2ec-43b3-b64a-14fde5e25efd">

1. It could get a bit wider to fill the remaining blank space, no? Not a grid expert, though.
2. The vertical whitespace looks pretty bad.

Reply:
I noticed that 1 might be because I don't have your version of instance views.